lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,115 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: string
+      )
+    end
+
+    def analyzer_version
+      Runners::VERSION
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      Results::Success.new(
+        guid: guid,
+        analyzer: analyzer,
+        issues: changes.changed_paths.map { |path| generate_issue(path) }
+      )
+    end
+
+    private
+
+    def generate_issue(path)
+      filepath = relative_path(path)
+      loc = text_file?(filepath.to_path) ? analyze_line_of_code(filepath.to_path) : nil
+      last_commit = analyze_last_commit_datetime(filepath.to_path)
+
+      Issue.new(
+        path: filepath,
+        location: nil,
+        id: "metrics_fileinfo",
+        message: "#{filepath}: loc = #{loc}, last commit datetime = #{last_commit}",
+        object: {
+          line_of_code: loc,
+          last_commit_datetime: last_commit
```

Comment:
[IMO] We often use naming like `*ed_at` according to Rails:

https://github.com/sider/runners/blob/0dc3eafffd38bba59d9fbb8394ebb5aa47eb4ff9/lib/runners/schema/trace.rb#L6

So it sounds readable. But this is my personal opinion, you can ignore it.

```suggestion
          last_committed_at: last_commit
```

Reply:
I should respect conventions of project. I've fixed them. 
fix: f9e857750285da53ce815047aad35c9e0f5eb781 and 75757e08bd8622181635335f88c6716066854000

@suetsuguk I've changed  `object` schema in `Issue` class as above. Please check that. 🙏

Reply:
Confirmed. Changes at this moment doesn't affect Sideci.